samples/contact_time.py

```diff
@@ -76,4 +66,6 @@
 # Get the contact time calculated on the fly
 contact_times = contact_time_accumulator.contact_times()
 
-print(f"The contact times are {contact_times}")
+print("The contact times are:")
+for pid, tau in zip(ids, contact_times):
+    print(f"  particle {pid}: {tau:.3f}")
```

Comment:
I find this  misleading. `contact_times`  contains tracks all the contact times measured in the simulation so it does not make sense to link it to any specific particle id.

Reply:
My bad, I'll fix it.